### PR TITLE
Feature/1447 ngsi orion sink create avega

### PR DESCRIPTION
**Pull request by AlvaroVega**
and close PRs: https://github.com/telefonicaid/fiware-cygnus/pull/1504 and https://github.com/telefonicaid/fiware-cygnus/pull/1480

_edit_: implements issue #1447

### CODE REVIEW

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java [hunk 1]
```diff
@@ -131,30 +130,27 @@ public Event intercept(Event event) {
...
             } // for
- 
+
             return interceptedEvents;
         } // if else
     } // intercept
 
     @Override
     public void close() {
         /*
```
Comment (fgalan) [inline cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java]: I understand this is a leftover a commneted code should be removed.

File: docker/cygnus-ngsi/Dockerfile [hunk 2]
```diff
@@ -154,12 +154,33 @@ ENV CYGNUS_POSTGRESQL_ENABLE_CACHE ""
...
+ENV CYGNUS_ORION_BATCH_TIMEOUT ""
+ENV CYGNUS_ORION_BATCH_TTL ""
+ENV CYGNUS_ORION_BATCH_RETRY_INTERVALS ""
+ENV CYGNUS_ORION_ENABLE_CACHE ""
+
 
 # NOTE: Configure correctly GIT_URL_CYGNUS and GIT_REV_CYGNUS for each git branch/fork used
 ENV GIT_URL_CYGNUS "https://github.com/telefonicaid/fiware-cygnus.git"
 # ENV GIT_URL_CYGNUS "https://github.com/myuser/fiware-cygnus.git"
-ENV GIT_REV_CYGNUS "master"
```
Comment (fgalan) [inline docker/cygnus-ngsi/Dockerfile]: This should be fixed. I understand that "master" should be the right one.

File: docker/cygnus-ngsi/agent.conf [hunk 3]
```diff
@@ -167,6 +166,30 @@ cygnus-ngsi.sinks.postgresql-sink.postgresql_password =
 #cygnus-ngsi.sinks.postgresql-sink.backend.enable_cache = false
 
 
+# NGSIOrionSink configuration
+cygnus-ngsi.sinks.orion-sink.type = com.telefonica.iot.cygnus.sinks.NGSIOrionSink
+cygnus-ngsi.sinks.orion-sink.channel = orion-channel
+# cygnus-ngsi.sinks.orion-sink.enable_encoding = false
```
Comment (fgalan) [inline docker/cygnus-ngsi/agent.conf]: If this is commented on purpose a comment line explainin why should be added.

File: docker/cygnus-ngsi/Dockerfile (hunk 2 above)
Comment (AlvaroVega) [reply docker/cygnus-ngsi/Dockerfile]: Fixed in a5a550e971e6179f2570362ec59ef17bb1475975

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java (hunk 1 above)
Comment (AlvaroVega) [reply cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java]: NTC: There is not a leftover, is just an empty line.

File: docker/cygnus-ngsi/agent.conf (hunk 3 above)
Comment (AlvaroVega) [reply docker/cygnus-ngsi/agent.conf]: fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1529/commits/a6681b7d4caffc733caeaa45915351e9c3f4a6e3

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java (hunk 1 above)
Comment (AlvaroVega) [reply cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java]: fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1529/files#diff-0b9888490f0a264991153957f9271deb

**Pull request merged**